doc/tests/tests_tutorial.rst

```diff
@@ -67,3 +67,21 @@ Kinematic Tests
 ---------------
 
 An additional test suite for the KinematicBase features in MoveIt! is available in the package `moveit_kinematic_tests <https://github.com/ros-planning/moveit_kinematics_tests>`_.
+
+
+Code Coverage
+-------------
+
+Test coverage measures the lines of code that are executed while running the test suite.
+To accumulate statistics and create a html coverage report, build the code without optimization
+and run the special `_coverage` target::
+
+  sudo apt-get install ros-melodic-code-coverage
+  catkin_make -DENABLE_COVERAGE_TESTING=ON -DCMAKE_BUILD_TYPE=Debug moveit_core_coverage
```

Comment:
Why `catkin_make`? The rest of the tutorials/documentation has all migrated to `catkin build` and we can't mix commands in a single workspace. Can we replace this with a `catkin build` command?

Reply:
yep. Done.